packages/pci-sdk-iframe/src/services/credentials.service.ts

```diff
@@ -0,0 +1,24 @@
+const API_KEY_KEY = '__apto__apk__';
+const USER_TOKEN_KEY = '__apto__ust__';
+
+function getAuth() {
+	const apiKey = sessionStorage.getItem(API_KEY_KEY);
+	const userToken = sessionStorage.getItem(USER_TOKEN_KEY);
```

Comment:
Do we need to store these values in `sessionStorage` ? 

Basically, I think [we're still vulnerable to XSS attacks ](https://blog.ropnop.com/storing-tokens-in-browser/#session-storage)in this way as a malicious script in the application of our client could get both credentials by looking into `sessionStorage`.

I think the safest here would be [to use a closure](https://blog.ropnop.com/storing-tokens-in-browser/#closure-variable), as mentioned in the article, as leaves malicious scripts without almost access to it.